.github/workflows/cron.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: Trigger nightly images
+
+on:
+  schedule:
+    # nightly packaging job usually finish around 14 UTC
+    - cron:  '0 15 * * *'
```

Comment:
One thing the CircleCI version I wrote did was to trigger devel and nightly tags at different times of the day, as to reduce the queuing time for the entirety of the nightly tags, so that they all finished together as ASAP.

Reply:
From my understanding of GH Action triggers, to do this we'll need 2 different workflows/yaml files. This is not hard to do though, and may be worth doing as the devel image can be built at any time of the day (see https://github.com/osrf/docker_images/pull/348/commits/c788ed32fe6dd7caf3e56b1aa4133201558adc04).

We'll still have some time between the various nightly tags though, and this can lead to inconsistencies for people running jobs using several of them during that period...

I guess that was an unexpected advantage of building them in the ci and pushing them.